thirdeye-detectionpipeline/src/main/java/ai/startree/thirdeye/detectionpipeline/operator/DelayOperator.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.detectionpipeline.operator;
+
+import static ai.startree.thirdeye.spi.util.SpiUtils.optional;
+
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class DelayOperator extends DetectionPipelineOperator{
+
+  private static final String DELAY_TIME = "delayTime";
+  private static final String RANDOM_OFFSET = "randomOffset";
+  @Override
+  public void execute() throws Exception {
+    long delayTime = Long.parseLong(getPlanNode().getParams().get(DELAY_TIME).toString());
+    final AtomicInteger randomOffset = new AtomicInteger();
+    optional(getPlanNode().getParams().get(RANDOM_OFFSET))
```

Comment:
I don't quite understand this random offset thing. What is the requirement here?

Reply:
The requirement is to provide a way to avoid having same task execution time for each execution and have something close to a real life behaviour.

Reply:
As discussed, let's remove the random offset feature.

Reply:
done